DBTableToCDAPTable/README.rst

```diff
@@ -41,7 +51,7 @@ Configurations for the Database Table Source
 
 #. ``splitBy``: The is the field used to generate splits. Not required if numSplits is set to one.
 
-#. ``numSplits``: The number of splits to generate.
+#. ``numSplits``: A string of the number of splits to generate (optional).
```

Comment:
I don't think we ever explicitly say that. JSON itself does not require strings, but do we in _all_ our JSON?

If so, somewhere we should say that.